robot-server/tests/integration/http_api/runs/test_run_queued_protocol_commands.tavern.yaml

```
@@ -0,0 +1,195 @@
+test_name: Make sure empty run commands execute protocol commands queue
+
+marks:
+  - usefixtures:
+      - run_server
```

Since #12139, we need `clean_server_state` too, to make sure this test's changes to the server don't leak into other tests.

```suggestion
      - run_server
      - clean_server_state
```